### PR TITLE
feat: Print the current context when chectl is run

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Prints the current context when chectl is run

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15741

### CODE REVIEW

**Comment by tolusha** [comment]
[test]

File: src/api/kube.ts [hunk 1]
```diff
@@ -54,7 +61,7 @@ export class KubeHelper {
   }
 
   async deleteAllServices(namespace = '') {
-    const k8sApi = this.kc.makeApiClient(CoreV1Api)
+    const k8sApi = KubeHelper.KUBE_CONFIG.makeApiClient(CoreV1Api)
```
Comment (mmorhun) [inline src/api/kube.ts:64]: What if we need to have different configs? For example make queries from different accounts?

Comment (tolusha) [reply src/api/kube.ts:64]: I have not faced such a case

**Comment by tolusha** [comment]
After digging into the code I figured out that `KubeConfig` already takes into account `KUBECONFIG` env var

```javascript
        if (process.env.KUBECONFIG && process.env.KUBECONFIG.length > 0) {
            const files = process.env.KUBECONFIG.split(path.delimiter);
            this.loadFromFile(files[0]);
            for (let i = 1; i < files.length; i++) {
                const kc = new KubeConfig();
                kc.loadFromFile(files[i]);
                this.mergeConfig(kc);
            }
            return;
        }
```

File: src/api/kube.ts (hunk 1 above)
Comment (mmorhun) [reply src/api/kube.ts:59]: But we may face it one day...

Comment (tolusha) [reply src/api/kube.ts:59]: Could you elaborate more?

**Comment by tolusha** [comment]
[test]

File: src/api/kube.ts [hunk 2]
```diff
@@ -28,19 +28,21 @@ import { V1alpha2Certificate } from './typings/cert-manager'
 const AWAIT_TIMEOUT_S = 30
 
 export class KubeHelper {
-  kc = new KubeConfig()
-  portForwardHelper = new PortForward(this.kc, true)
-  logHelper = new Log(this.kc)
+  public static readonly KUBE_CONFIG = KubeHelper.initializeKubeConfig()
```
Comment (mmorhun) [inline src/api/kube.ts:31]: I am not keen on the idea to make kubeconfig static.

Comment (tolusha) [reply src/api/kube.ts:31]: Why?

Comment (mmorhun) [reply src/api/kube.ts:31]: `KubeHelper` is helper class which is used to communicate with Kubernetes API. By making kubeconfig static you basically make whole class static, so it becomes more a namespace than a class... It is my vision...

File: src/api/kube.ts (hunk 1 above)
Comment (mmorhun) [reply src/api/kube.ts:59]: I mean not this specific line of code, I am talking about more general stuff. Maybe we do not need it at all (as we don't right now), but static kubeconfig makes chectl somewhat limited.

Comment (tolusha) [reply src/api/kube.ts:59]: It is absolutely not true, it brings consistency. Image the case when `chectl` is working and meanwhile user changed the context? Then a new instance of KubeHelper will use a kubeconfig with a different context that might lead to errors.

**Comment by tolusha** [comment]
[test]

**Pull request merged**